pulp_ansible/app/galaxy/views.py

```diff
@@ -20,6 +20,46 @@
 )
 
 
+class DistributionMixin:
```

Comment:
This looks a lot like the `galaxy.v3.views.AnsibleDistributionMixin`. Only that the other one adds signatures. Right?

Reply:
Correct, it's been copied pasted over.